test/system/roles/public_health/public_health_custom_export_test.rb

```diff
@@ -89,6 +95,10 @@ def all_custom_export_assessment_fields
         laboratories: {
           selected: ['Lab Results']
         },
+        vaccines: {
+          selected: ['Vaccinations'],
+          checked: all_custom_export_vaccine_fields
```

Comment:
I think this can just be replaced with `VACCINE_FIELD_NAMES.keys` (same for the other data types), and then all those helper methods can be removed

Reply:
I think we need to leave `all_custom_export_patient_fields`, since those options have nested fields. But I agree for `assessment` and `vaccine` it looks like having helpers is overly complicated.